test/tests/test_generate_test_runner.rb

```diff
@@ -1158,9 +1158,43 @@
       :to_pass => [ ],
       :to_fail => [ ],
       :to_ignore => [ ],
-      :text => [ "ERROR: Unknown Option z" ],
+      :text => [
+        "ERROR: Unknown Option z",
+        "Options:",
+        "-l        List all tests",
+        "-f TEST   Only run tests with TEST in the name",
+        "-n TEST   Only run tests with TEST in the name",
```

Comment:
see above

Reply:
Marked as depreciated. I can remove it if you'd prefer.